.github/workflows/ci.yml

```
@@ -19,9 +19,9 @@ jobs:
         uses: pnpm/action-setup@v2
 
       - name: Set node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: 16.x
+          node-version: latest
```

```suggestion
          node-version: lts/*
```

Maybe lts would be a better target

---

Sure! I willupdate the typecheck action below as well.